src/cmd/vm.rs

```diff
@@ -52,7 +52,7 @@ pub struct ArgsSetup {
     /// path to dir where betty.sh exists. If omitted, current directory will be
     /// used.
     #[argh(option)]
-    repo: Option<String>,
+    arc: Option<String>,
```

Comment:
The `--repo` option is also used when specifying cros directory, which is confusing, so I changed it to the `--arc` option.

Reply:
That's better but I'd think I'd prefer --arc_repo or --android_repo

Reply:
Sorry, I didn't explain the context. I heard that the `--repo` option indicating the ChromeOS repository will be renamed to --cros in the future, so I changed it to `--arc` accordingly.

CC @hikalium

Reply:
Hmmmm i guess I think now both the names are bad but ok.